### PR TITLE
Example of using Avro serializer with generated (specific) Avro objects

### PR DESCRIPTION
**Pull request by gwenshap**
It is super similar to what already exists in producer example, but using generated Avro objects.

### CODE REVIEW

File: specific-avro-producer/src/main/resources/avro/LogLine.avsc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "name": "LogLine",
+  "fields": [
+    {"name": "ip", "type": "string"},
+    {"name": "timestamp",  "type": "long"},
+    {"name": "url",  "type": "string"},
+    {"name": "referrer",  "type": "string"},
+    {"name": "useragent",  "type": "string"},
+    {"name": "sessionid",  "type": ["null","int"], "default": null}
+  ]
+}
```
Comment (Ishiihara) [inline specific-avro-producer/src/main/resources/avro/LogLine.avsc]: empty line here.

File: specific-avro-producer/pom.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <finalName>uber-${artifactId}-${version}</finalName>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (Ishiihara) [inline specific-avro-producer/pom.xml]: empty line at the end

**Comment by Ishiihara** [comment]
LGTM. The inline commends are mainly formatting issues.

**Comment by gwenshap** [comment]
Do I really need to sign the CLA? I already signed my IP away when I joined :)

**Comment by hachikuji** [comment]
add to whitelist

**Comment by ewencp** [comment]
@hachikuji "add to whitelist" is for the PR builder on Jenkins, which I don't think we have setup for this repo yet (partly because this repo is rarely updated anyway and doesn't even have unit tests). The CLA bot is separate. It wasn't properly ignoring collaborators (people we've given push access to the repos, which means Confluent employees). I think I've now fixed that issue so it should stop prompting us now.

**Comment by gwenshap** [comment]
Now that my CLA was sorted out and @Ishiihara comments addressed, do I merge it myself, or do I need one of you to merge this?

**Comment by Ishiihara** [comment]
@gwenshap You can go ahead and merge it.

**Comment by hachikuji** [comment]
@ewencp Haha, I just knew that as one of the magic incantations that you gave for new users to make things possible on github, so I thought I'd give it a try. Incidentally, can we change the phrase to "open sesame"?

**Comment by ewencp** [comment]
@hachikuji Haha, good idea. It's a regex in the global config for the Jenkins plugin, so we can change it to match a variety of phrases.

**Pull request merged**